Test/Pester5.Tests.ps1

```diff
@@ -278,8 +137,25 @@ Describe 'One' {
     }
 }
 '@
-        WhenPesterTaskIsInvoked -AsJob
-        ThenPesterShouldHaveRun -FailureCount 0 -PassingCount 2
+        WhenPesterTaskIsInvoked -AsJob -WithArgument @{
+            Debug = @{
+                ShowFullErrors = ($DebugPreference -eq 'Continue');

```

Comment:
Unless you're testing that these properties are getting set/passed to Pester, you can omit them. Only include properties that are part of your test. In this case, I think just `Run.Path` is necessary since you're only testing that the task passed.